File: Views/src/components/financial/dto/checking-account-dto.ts
import { CardDto } from "../components/card/dto/card-dto";

export class CheckingAccountDto {
  id:number;
  institution:string;
  holder:string;
  agency:number;
  account:number;
  manager:string;
  pix:string;
  typeaccount:string;
  cards: CardDto[];
  description:string;
}

File: Services/Dto/Financial/CheckingAccountDto.cs

using System.Collections.Generic;

namespace Services.Dto.Financial
{
    public class CheckingAccountDto
    {
        public int Id { get; set; }
        public string Institution { get; set; }
        public string Holder { get; set; }
        public string Agency { get; set; }
        public string ManagerName { get; set; }
        public string ManagerContact { get; set; }
        public string Account { get; set; }
        public string Pix { get; set; }
        public decimal Balance { get; set; }
        public string Typeaccount { get; set; }
        public List<CardDto> Cards { get; set; }
        public string Description { get; set; }
    }
}